sql/src/test/java/io/crate/external/S3ClientHelperTest.java

```diff
@@ -24,18 +24,31 @@
 import com.amazonaws.http.IdleConnectionReaper;
 import com.amazonaws.services.s3.AmazonS3;
 import io.crate.test.integration.CrateUnitTest;
+import org.elasticsearch.common.settings.MockSecureSettings;
+import org.elasticsearch.common.settings.Settings;
 import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 
 import java.net.URI;
 import java.net.URL;
 import java.util.Date;
 
+import static io.crate.external.S3ClientHelper.ACCESS_KEY_SETTING_NAME;
+import static io.crate.external.S3ClientHelper.SECRET_KEY_SETTING_NAME;
 import static org.hamcrest.Matchers.is;
 
 public class S3ClientHelperTest extends CrateUnitTest {
 
-    private final S3ClientHelper s3ClientHelper = new S3ClientHelper();
+    private S3ClientHelper s3ClientHelper;
+
+    @Before
+    public void setupS3ClientHelper() {
+        MockSecureSettings secureSettings = new MockSecureSettings();
+        secureSettings.setString(ACCESS_KEY_SETTING_NAME, "testS3AccessKey");
+        secureSettings.setString(SECRET_KEY_SETTING_NAME, "testS3SecretKey");
+        s3ClientHelper = new S3ClientHelper(Settings.builder().setSecureSettings(secureSettings).build());
```

Comment:
shall we add a test for anonymous mode?

Reply:
@andrei fixup for this, also using new way and not the deprecated `AmazonS3Client()` construnctors.